api/server/publications.ts

```diff
@@ -24,34 +28,33 @@ export function initPublications() {
     return Messages.collection.find({
       chatId
     }, {
-      sort: {createdAt: -1},
+      sort: { createdAt: -1 },
       limit: 30 * messagesBatchCounter
     });
   });
 
-  Meteor.publishComposite('chats', function() {
+  Meteor.publishComposite('chats', function(): PublishCompositeConfig<Chat> {
```

Comment:
Why use publish composite? I think that this is something you can do with RxJS Operators to merge the data streams in the client side instead of creating a complicated publication.